scripts/aarch64-libc2_17-cross-build.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+## NOTE:
+## This script must be run in the centos:7 container which is similar to "manylinux2014_aarch64"
+## and has the same toolchain version / glibc / kernel / ABI.
+
+set -e
+
+# install required cross compiler/linker
+yum install -y dnf epel-release gcc curl
+yum install -y binutils-aarch64-linux-gnu gcc-aarch64-linux-gnu gcc-c++-aarch64-linux-gnu
+
+# install Rust toolchain
+curl https://sh.rustup.rs -sSf | sh -s -- -y
+source "$HOME/.cargo/env"
+# install our target
+rustup target add aarch64-unknown-linux-gnu
+
+# since the Centos is quite old, and were are on the different architecture, we will disable gpg verification for now
```

Comment:
I don't really understand why we disable gpg verification. Is the reason to not do too many things for now? It's ok to do that, but I don't get why working on an old version influences the decision.

Reply:
Since the distribution is very old and the script also installs binaries from completely different architecture, the gpg check fails very often. 
Another way would be to introduce some retries/backoffs for fetching keys from the mirrors.